eventsourcing/postgres.py

```diff
@@ -338,6 +341,8 @@ def __init__(self, application_name: str):
                 f"'{self.POSTGRES_HOST}'"
             )
 
+        port = self.getenv(self.POSTGRES_PORT, "5432")
```

Comment:
Hi @poltronSuperstar! Thanks so much for making these changes.

To satisfy mypy, this needs to be:
```
port = self.getenv(self.POSTGRES_PORT) or "5432"
```

Then it's good to go :) and I will merge.

Reply:
Cool :) (I find the mypy compliant more readable anyway !)
By the way thanks for that lib, totally amazing and a pleasure to use.

Reply:
So glad you like it! :) thanks for making this change so quickly. If there's any thing else, don't hesitate...

Reply:
Oh while I'm here, I have a question, if you have 3 minutes to answer, otherwise it's really not that important.
```python
class Factory(InfrastructureFactory):
    POSTGRES_DBNAME = "POSTGRES_DBNAME"
```
Is there other reason to use class attr instead of literal than complying with linters ?
I would generally agree that magic values are cancer, but in this particular case, I tend to use literal strings directly as it's really functionally similar to using a constant.